internal/gps/source.go

```diff
@@ -65,8 +65,6 @@ func newSourceCoordinator(superv *supervisor, deducer deducer, cachedir string,
 	}
 }
 
-func (sc *sourceCoordinator) close() {}
```

Comment:
let's leave this in, please. we can add a comment about why we're preserving the no-op func, but i would rather have the call to it remain down in `doRelease()` so that, if we _do_ have cleanup that we need to add later, we've already got the right hook in place.